dataland_analytics/analysis_scripts/scripts_with_api_read/datasets_per_framework_script.py

```diff
@@ -0,0 +1,22 @@
+import dataland_datasets
+from dataland_analytics.config import *
+import itertools
+import pandas as pd
+
+configuration = dataland_datasets.Configuration(
+	host = "https://dataland.com/api",
+	access_token=PERSONAL_USER_API_KEY
```

Comment:
Bei mir funktionert das irgendwie nicht. Obwohl der token in einem curl funktioniert bekomme ich ein "Unauthorized 401" Fehler wenn ich das Skript laufen lasse.

Reply:
Wie telefonisch besprochen gelöst. Resolving.